modules/virt-example-vf-host-services.adoc

```diff
@@ -59,12 +56,11 @@ spec:
 <9> Identifies the VF with an ID of `0`.
 <10> Sets a maximum transmission rate, in Mbps, for the VF. This sample value sets a rate of 200 Mbps.
 
-The following YAML file is an example of a manifest that creates a VLAN interface on top of a VF and adds it to a bonded network interface.
-It includes samples values that you must replace with your own information.
+The following YAML file is an example of a manifest that adds a VF for a network interface. In this configuration, the `ens1f1v0` VF is created on the `ens1f1` physical interface, and this VF is added to a bonded network interface `bond0`. The bond uses `active-backup` mode for redundancy. In this example, the VF is configured to use hardware offloading to manage the VLAN directly on the physical interface.
```

Comment:
Another way of doing this to break up the information that is a bit dense is to move everything after first sentence to after the code block as an explanation of it. My next comment has an alternative method as well.

Reply:
Thanks for your points on this one Joe. I've added a line break after the first sentence. And I removed the line that says to replace with your own values. I also "In this sample configuration..." now. 

The callouts describe the fields, rather than the relationship between the fields in this particular example. As there's quite a lot of inter-dependent networking going on here, I think a single para describing it as a whole is worthwhile.